packages/tiled/lib/src/tile_map_parser.dart

```diff
@@ -10,12 +10,19 @@ class TileMapParser {
   ///
   /// Accepts an optional list of external TsxProviders for external tilesets
   /// referenced in the map file.
-  static TiledMap parseTmx(String xml, {List<TsxProvider>? tsxList}) {
+  static TiledMap parseTmx(
+    String xml, {
+    List<TsxProvider>? tsxList,
+    Future<TsxProvider> Function(String key)? tsxProviderFunction,
```

Comment:
what's the difference between these two? seems like two different ways to pass in external tsx files?